### PR TITLE
Adding DPIF-R3 config

### PR DESCRIPTION
**Pull request by albertkol**
### How to test
1. Be on latest docker-runner and all other repos
2. Git checkout `run-dpif-r3` branch on:
- digital-form-builder-adapter
- funding-service-pre-award-stores
- funding-service-design-frontend
- funding-service-design-notification
3. `make up`
4. Annoying extra step because open date is in the future:
```
docker exec funding-service-pre-award-stores-1 python -m fund_store.scripts.amend_round_dates -q update-round-dates-fab --fund_short_name DPIF --round_short_name R3 -o PAST -d FUTURE
```
5. Go to https://frontend.levellingup.gov.localhost:3008/funding-round/DPIF/R3

### CODE REVIEW

File: config/envs/default.py [hunk 1]
```diff
@@ -84,6 +84,7 @@ class DefaultConfig:
 
     # E.G. "EMAIL": "GOV_NOTIFY_ID"
     REPLY_TO_EMAILS_WITH_NOTIFY_ID = {
+        "digitalplanningteam@communities.gov.uk": "73eecbb1-5dbc-4653-8c58-46aa79151210",
```
Comment (gidsg) [inline config/envs/default.py:87]: This is already in the list? Line 93 so think PR is not needed?

**Review by nuwan-samarasinghe — DISMISSED** [review]
LGTM

File: config/envs/default.py (hunk 1 above)
Comment (gidsg) [inline config/envs/default.py:87]: This ID is listed twice so I guess one of them should be removed?

**Review by gidsg — DISMISSED** [review]

File: config/envs/default.py (hunk 1 above)
Comment (srh-sloan) [reply config/envs/default.py:87]: You are right. But I've now edited the email direct on notify so there's just one email address, `digitalplanningteam@communities.gov.uk` with the same existing ID `73eecbb1-5dbc-4653-8c58-46aa79151210`, so this PR can be abandoned

Comment (srh-sloan) [reply config/envs/default.py]: Also, for future reference, we can have 2 emails in this dict with the same ID, but the actual `reply-to` email that gets set from notify will be what's in notify, not whatever email we have here. So with this PR the actual `reply-to` on emails sent from notify would be `@levellingup.gov.uk`

**Review by srh-sloan — APPROVED** [review]

File: config/envs/default.py (hunk 1 above)
Comment (albertkol) [reply config/envs/default.py]: Updated the domain to `communities.gov.uk`, so we prevent adding an extra line.

**Pull request merged**